plant-and-cell-physiology.csl

```diff
@@ -0,0 +1,227 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" et-al-min="6" et-al-use-first="6" demote-non-dropping-particle="sort-only" default-locale="en-GB" xmlns="http://purl.org/net/xbiblio/csl">
```

Comment:
Yeah, the spec could probably be made a little clearer, but "use-first" must by definition be lower than "min". "min" defines the minimum number of authors at which et-al abbreviation kicks in, and because "et al." means "and others", you must always print fewer names than that.